docs/en/inference/pipeline.md

```diff
@@ -6,29 +6,47 @@ Then, we will describe the pipeline API in detail.
 
 ## Usage
 
-An example using default parameters:
+- An example using default parameters:
 
 ```python
 from lmdeploy import pipeline
 
-pipe = pipeline('internlm/internlm-chat-7b')
+pipe = pipeline('internlm/internlm2-chat-7b')
 response = pipe(['Hi, pls intro yourself', 'Shanghai is'])
 print(response)
 ```
 
-An example showing how to set tensor parallel num:
+In this example, the pipeline by default allocates 50% of the GPU total memory for k/v caches.
+Out Of Memory (OOM) errors may occur if a 7B model is deployed on a GPU with memory less than 40G.
+If you encounter an OOM error, please decrease the ratio of the k/v cache occupation as follows:
+
+```python
+from lmdeploy import pipeline, TurbomindEngineConfig
+
+# decrease the ratio of the k/v cache occupation to 20%
+backend_config = TurbomindEngineConfig(cache_max_entry_count=0.2)
+
+pipe = pipeline('internlm/internlm2-chat-7b',
+                backend_config=backend_config)
+response = pipe(['Hi, pls intro yourself', 'Shanghai is'])
+print(response)
+```
+
+A better approach would be to allocate space for the k/v cache from the free GPU memory proportionally. We will improve it in future versions for a better user experience.
```

Comment:
It has been merged. So we may update this description. Thanks.

Reply:
Sure

Reply:
Updated. Please review again